### ISSUE TITLE
Missing Icons

### ISSUE THREAD

**Issue by askedrelic** [rhydlewis/search-omnifocus#8]
Hey, thanks for this plugin, I think it's a great idea!

I can't get the default icons for tasks, as shown in the README, to show up. Any idea what could be going on or are icons something that always need to be manually setup?

Here is ".lc" search: http://asktherelic.com/priv/20150912-1116-8o9m6.png
The default icons are working though... http://asktherelic.com/priv/20150912-1117-s551y.png

This is with Alfred 2 latest (2.7.2) and your plugin latest (v1.1.9)

**Comment by rhydlewis** [comment]
The workflow assumes that you've installed Omnifocus in this location:

    /Applications/OmniFocus.app/Contents/Resources

If you open Terminal.app and type:

    ls -l /Applications/OmniFocus.app/Contents/Resources/dropped@2x.png

what does it show you? For me it shows:

<img width="757" alt="screenshot 2015-09-12 13 49 12" src="https://cloud.githubusercontent.com/assets/2102276/9831554/190d2c2c-5955-11e5-9756-12d94efa4d24.png">

However, if you see:

    ls: /Applications/OmniFocus.app/Contents/Resources/dropped@2x.png: No such file or directory

then this means Omnifocus isn't installed in the usual place and I'll need to update the workflow.

Let me know!

**Comment by askedrelic** [comment]
Ah indeed, I had run into some issues with previous versions of the script not working 100%, which I saw in the changelog were because of filepaths.

I'm using MacStore purchased Omnifocus, which seems to install in :
```
ls -l /Applications/OmniFocus.localized/OmniFocus.app/Contents/Resources/dropped@2x.png
```

I made a copy of Omnifocus to /Applications/ and icons work now! I might recommend check both places for icons then. Thanks for the help!
